pulp_docker/app/viewsets.py

```diff
@@ -148,42 +149,40 @@ def sync(self, request, pk):
         return OperationPostponedResponse(result, request)
 
 
-class DockerPublisherViewSet(PublisherViewSet):
+class DockerPublicationViewSet(NamedModelViewSet, mixins.CreateModelMixin):
```

Comment:
I think this should probably also have these mixins: [list, read, delete]

Reply:
just create, this is a custom endpoint